### ISSUE TITLE
update to zbus v4

### ISSUE THREAD

**Issue by thewh1teagle** [hwchen/keyring-rs#166]
I use dialog plugin from [tauri](https://github.com/tauri-apps/tauri) which uses zbus v4,
And I can't use this library along with that plugin because there's a conflict.

Do you have plans to update it to zbus v4? [secret-service/Cargo.toml#L37](https://github.com/hwchen/secret-service-rs/blob/master/Cargo.toml#L37)

Error:

```console
error: failed to select a version for `zvariant_utils`.
    ... required by package `zbus_macros v4.0.0`
    ... which satisfies dependency `zbus_macros = "=4.0.0"` of package `zbus v4.0.0`
    ... which satisfies dependency `zbus = "^4.0"` of package `ashpd v0.8.0`
    ... which satisfies dependency `ashpd = "^0.8"` of package `rfd v0.14.0`
    ... which satisfies dependency `rfd = "^0.14"` of package `tauri-plugin-dialog v2.0.0-beta.1 (https://github.com/thewh1teagle/plugins-workspace?branch=v2#dc6d3321)`
    ... which satisfies git dependency `tauri-plugin-dialog` of package `app_name v0.2.8 (/Users/user/Documents/app_name/app_name/src-tauri)`
versions that meet the requirements `=1.1.0` are: 1.1.0

all possible versions conflict with previously selected packages.

  previously selected package `zvariant_utils v1.0.1`
    ... which satisfies dependency `zvariant_utils = "=1.0.1"` (locked to 1.0.1) of package `zbus_macros v3.15.2`
    ... which satisfies dependency `zbus_macros = "=3.15.2"` (locked to 3.15.2) of package `zbus v3.15.2`
    ... which satisfies dependency `zbus = "^3.7"` (locked to 3.15.2) of package `secret-service v3.0.1`
    ... which satisfies dependency `secret-service = "^3"` (locked to 3.0.1) of package `keyring v2.3.2`
    ... which satisfies dependency `keyring = "^2.2.0"` (locked to 2.3.2) of package `app_name` v0.2.8 (/Users/user/Documents/app_name/app_name/src-tauri)`
```

**Comment by brotskydotcom** [comment]
I have no plans to update to being dependent on v4 of zbus, but if it's API compatible with v3 I'm happy to allow it in the dependency tree.  I will try this out locally to see if it works.

**Comment by brotskydotcom** [comment]
Sorry, but I'm not a secret-service maintainer, so I can't help you with this.  I see you've also filed a bug with secret-service about this; hopefully they will get to it fairly soon and then I will take their update here.

**Comment by brotskydotcom** [comment]
The tracking bug is hwchen/secret-service-rs#71, which was closed before the release was made.  Hoping to see a release soon.

**Comment by brotskydotcom** [comment]
The secret-service release has been made, and I'll get it integrated in the next week or two. I'm leaning towards making this a minor release, since the API for this crate is not changing, but I'm open to thoughts from others.  It's such a big dependency change it might be worth a major release just to warn people.

**Comment by brotskydotcom** [comment]
Fixed in PR #179. Please try the pre-release.
